src/course-unit/clipboard/paste-component/components/WhatsInClipboard.jsx

```diff
@@ -0,0 +1,58 @@
+import { useRef } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Icon } from '@edx/paragon';
+import { Question as QuestionIcon } from '@edx/paragon/icons';
+
+import messages from '../messages';
+
+const WhatsInClipboard = ({
+  handlePopoverToggle, togglePopover, popoverElementRef,
+}) => {
+  const intl = useIntl();
+  const triggerElementRef = useRef(null);
+
+  const handleKeyDown = ({ key }) => {
+    if (key === 'Tab') {
+      popoverElementRef.current.focus();
+      handlePopoverToggle(true);
+    }
+  };
+
+  return (
+    <div
+      className="paste-component-whats-in-clipboard mt-2 d-flex align-items-center"
+      data-testid="whats-in-clipboard"
```

Comment:
[nit] the same name as class above

Reply:
[nit] please check and below too

Reply:
Thanks! I decided to remove the prefix "paste-component" and take the name of the component as a basis